config/context/spring-context.xml

```diff
@@ -40,8 +40,9 @@
     <constructor-arg index="1" ref="keyPair"/>
     <constructor-arg index="2" ref="irohaAPI"/>
     <constructor-arg index="3" value="${quorum.key}"/>
-    <constructor-arg index="4" value="${accounts.holder}"/>
-    <constructor-arg index="5" value="${credential.accountId}"/>
+    <constructor-arg index="4" value="${brvs.userDomains}"/>
+    <constructor-arg index="5" value="${accounts.holder}"/>
+    <constructor-arg index="6" value="${credential.accountId}"/>
```

Comment:
Can we use arg `name` instead of `index`? Looks confusing.

Reply:
Let's do it in another PR